### ISSUE TITLE
Issue with ian:accounts-ui-bootstrap-3 

### ISSUE THREAD

**Issue by chieina** [okgrow/meteor-persistent-session#43]
after a meteor update, I got an issue with the accounts popup. I found out that the cause of the problem is the u2622:persistent-session. This is how it looks like :  

![accounts-bootstrap](https://cloud.githubusercontent.com/assets/3444151/9212934/d5bf786a-40bf-11e5-8273-82d69ed21803.jpg)

Thank you in advance for the response

**Comment by rgould** [comment]
@chieina Thanks! We'll look into it as soon as possible.

**Comment by chieina** [comment]
thanks in advance.

**Comment by dpatte** [comment]
what was the last working version installed before this appeared?

**Comment by rgould** [comment]
@chieina @dpatte Sorry, only just getting to this now. Unfortunately I can't seem to replicate it. I'm not sure if it's because I'm using a newer version of `ian:accounts-ui-bootstrap-3` (1.2.81) or because there's some other configuration involved.

What `accounts-*` packages are you using? I tried it with `accounts-password`, `accounts-google`, and `accounts-facebook`, and they all looked good.

If you're still experiencing the problem can you provide your `.meteor/packages`, and `.meteor/versions` files, and any `ian:accounts-ui-bootstrap-3` configuration you're using?

Thanks!

**Comment by dpatte** [comment]
I believe this is an issue with another package (meteor-persistant-session) as it is now resolved for me when I backtracked on that module to v0.3.5. @chieina - do you see the same?

**Comment by rgould** [comment]
@dpatte Now that I think of it there may have been a bug with 0.4.0 that was resolved in 0.4.3, which is the current release. Are you able to try that out and see if the problem persists?

**Comment by dpatte** [comment]
i upgraded from 0.3.5 to 0.4.4 and still see this issue. I am currently stuck at 0.3.5

**Comment by dpatte** [comment]
OK. I upgraded from 0.3.5 to 0.4.4 and still saw this issue (as well as other persistance issues). I was able to resolve the problems by clearing the browser cookies manually. All seems good now.

But I am trying to understand what has caused the issue, and if there is anything I can do that can fix it from this end for my clients. Expecting them to guess that they have to clear their cookies is something I'd like to try to avoid.
